scripts/publish/publish.ps1

```diff
@@ -135,6 +135,17 @@ function UploadInstallers($msiFile)
     return 0
 }
 
+function UploadVersionBadge($badgeFile)
+{
+    $fileName = [System.IO.Path]::GetFileName($badgeFile)
+    
+    Write-Host "Uploading the version badge to Latest"
+    UploadFile "dev/Binaries/Latest/$filename" $badgeFile
+    
+    Write-Host "Uploading the version badge to $env:DOTNET_CLI_VERSION"
+    UploadFile "dev/Binaries/$env:DOTNET_CLI_VERSION/$filename" $badgeFile
```

Comment:
The badge file name is always called `version_badge.svg` irrespective of which build (windows or osx or ubuntu or centos) is publishing them. So the badge will always get overwritten by the build that finishes last. We will definitely hit issues were one of the build fails but the badge number is still published by another successful build.

I suggest uploading 4 different badges for the 4 OSes we support.